project.h

```diff
@@ -92,7 +92,14 @@
 #define DEBUG_DEVICE_IO
 
 #define RT_PRIO_UART    39
```

Comment:
It seems `RT_PRIO_UART` is obsolete on the serdev branch.

Reply:
Hmm, this (and RT_PRIO_GATE) is indeed obsolete but is not part of the fix. Should it be in this patch nevertheless?